client/layout/logged-out.jsx

```diff
@@ -31,18 +31,13 @@ const LayoutLoggedOut = ( {
 
 	let masterbar = null;
 
-	if ( useOAuth2Layout ) {
-		const hasValidOAuth2ClientData = !! oauth2Client;
-		const oauthClientName = hasValidOAuth2ClientData && oauth2Client.name;
-		classNameObject.dops = hasValidOAuth2ClientData;
-		classNameObject[ oauthClientName ] = hasValidOAuth2ClientData;
+	// Uses custom styles for DOPS clients - which are the only ones with the name property defined
+	if ( useOAuth2Layout && oauth2Client && oauth2Client.name ) {
```

Comment:
`useOAuth2Layout` is only `true` if `oauth2Client` is defined, so the extra check isn't necessary.

Reply:
That's what I thought as well but after giving it another look that's not entirely true.

`useOAuth2Layout()` will be `true` if `ui.oauth2Clients.currentClientId` exists, which happens as soon as we provide this id [in the url](https://github.com/Automattic/wp-calypso/blob/dd8c833c11d7b0c5fbb7ceedae6528a988269a63/client/state/ui/oauth2-clients/reducer.js#L15). It doesn't mean that the corresponding client data at `oauth2Clients` (in the global state tree) is available though (since we need to fetch it).

Reply:
Oh, good catch! I totally missed that.